source/includes/_page_security.md

```diff
@@ -0,0 +1,473 @@
+# Pages security
+
+The page security is the set of rules that defines who (which users / groups) can explore or edit a given page.
+
+It is defined through 2 variables:
+
+* a page restriction through the `restricted` attribute in [the page object](#the-page-object), which define if a page is visible to every user who can explore the portal
+* specific rulesets for users and groups
+
+If the page is set as restricted, then the page will only appear in the portal for users who have a ruleset declared for them, either directly or through a group. Other users won't have any access to the page.
+Rulesets can also give users permission to edit the page and manage its security.
+
+### The page security API in a nutshell
+
+#### User level page security ruleset
+
+`/pages/{PAGE_SLUG}/security/users`
```

Comment:
Do users know what a "slug" is? Do we use this term somewhere in the UI?

Reply:
I don't think we use this term in the UI: In the UI we have "Page URL" label but the form input for the slug is prefixed by `/pages/` so it is more accurate.

In the Pages section, when I describe the Page object, I define the slug as "Human readable identifier used to generate the page's URL". Moreover, IMO an API documentation is for developers, and slug is a widely used term, almost official.

Do you still think I should document this more @juliendbg ?

Reply:
No that's fine like that 👍